docs/keys.md

```diff
@@ -1,4 +1,6 @@
-# Keys
+# Keys subcommand
```

Comment:
```suggestion
# Keys manager
```

Reply:
I think these sound better, what do you think?

Reply:
agreed! I'm leaning toward `Key management`